.github/workflows/cli.yml

```
@@ -83,3 +86,40 @@ jobs:
       #     status: ${{ job.status }}
       #     fields: job,message,ref,eventName,author,took
       #     author_name: Check packages
+  web:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: 👀 Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 100
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: ⬇️ Fetch commits from base branch
+        run: git fetch origin ${{ github.event.before || github.base_ref || 'main' }}:${{ github.event.before || github.base_ref || 'main' }} --depth 100
+        if: github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch'
+      - name: 🏗️ Setup Bun
+        uses: oven-sh/setup-bun@v1
+        with:
+          bun-version: latest
+      - name: Install Playwright Browsers
+        run: bun playwright install --with-deps
```

I think you are invoking the `playwright` dependency here, right? If so, 2 problems:

1. No package should be installed at this step yet, so invoking the bin probably won't work.
2. I see it's only added to `packages/@expo/cli`, if so, add the working directory so it's clear that that's where the dep is installed.

```suggestion
      - name: Install Playwright Browsers
        run: bun playwright install --with-deps
        working-directory: packages/@expo/cli
```

That won't fix the first issue though, either move this step after `🧶 Install node modules in root dir`, or use `bunx playwright@<version> install` -- but that won't be linked with the version from the package.json file.